src/lib/components/Breadcrumbs.svelte

```diff
@@ -18,6 +18,7 @@
     routeModules?: Record<string, ModuleData>;
     pageData: any;
     children?: Snippet<[any]>;
+    shouldSkipCrumbsForRoutesWithNoPage: boolean;
```

Comment:
Param name could be a bit shorter, `skipRoutesWithNoPage` maybe?

Reply:
sounds good! Changed it and merging this in now

Reply:
Sounds good! Changed the name and merging in now